top_five.py

```diff
@@ -0,0 +1,17 @@
+def get_top_five(n: dict):
```

Comment:
I'm still trying to figure out best practice for this sort of thing. I was thinking since main.py essentially gets the data for formats it into a dictionary of names and objects that maybe whatever is then done TO that data should be separate? Like if I decided to do a top ten, top fifteen, etc. shouldn't those options be in a separate module?

I guess I'm foggy on how MUCH code should be going into each .py file?

Reply:
honestly, just keep them all together until you have a reason to split them.  You'll see the natural split when you've got more than this.  Right now this is just small enough for it all to be in one